lambda/app/src/controllers/team.ts

```diff
@@ -205,22 +206,23 @@ export const requestServiceAccount = async (session: Session, userId: number, te
 
   await serviceAccount.save();
 
-  await sendTemplate(EMAILS.TEAM_API_SERVICE_ACCOUNT_REQUESTED, { requester, team, integrations });
+  await sendTemplate(EMAILS.CREATE_TEAM_API_ACCOUNT_SUBMITTED, { requester, team, integrations });
 
   return serviceAccount;
 };
 
 export const getServiceAccount = async (userId: number, teamId: number) => {
   const teamIdLiteral = getTeamIdLiteralOutOfRange(userId, teamId, ['admin']);
 
-  return models.request.findOne({
+  return await models.request.findOne({
```

Comment:
wonder any specific reason to add `await` here?

Reply:
Just to ensure the promise returned by the finder resolves

Reply:
Thanks for sharing that, removed the extra await.